src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,28 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        book.setTitle("second book");
+        book.setPrice(200);
+        Book bookFromDB = bookDao.create(book);
+        System.out.println(bookFromDB);
+
+        System.out.println("Has id = 4 deleted? - " + bookDao.deleteById(5L));
```

Comment:
Do you have book with id = 5 in DB?

Reply:
Yes I had, but after this method I don't have.